dot-gen/src/main/java/com/wipro/ats/bdre/ldg/LineageDotGen.java

```diff
@@ -17,6 +17,8 @@
 import com.wipro.ats.bdre.GetLineageQueryByProcessId;
 import com.wipro.ats.bdre.lineage.LineageConstants;
 import com.wipro.ats.bdre.lineage.LineageMain;
+import com.wipro.ats.bdre.md.api.GetProcess;
+import com.wipro.ats.bdre.md.beans.ProcessInfo;
 import com.wipro.ats.bdre.md.dao.jpa.LineageQuery;
 import org.apache.commons.cli.CommandLine;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.apache.commons.cli.CommandLine'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AUselessImportCheck)